geoportal/c2cgeoportal_geoportal/__init__.py

```diff
@@ -128,7 +130,17 @@ def add_admin_interface(config):
         )
         config.add_view(c2cgeoportal_geoportal.views.add_ending_slash, route_name="admin_add_ending_slash")
         config.add_route("admin_add_ending_slash", "/admin", request_method="GET")
-        config.include("c2cgeoportal_admin", route_prefix="/admin")
+        config.include("c2cgeoportal_admin")
```

Comment:
This is no more working with get it fixed?

Reply:
We made changes in c2cgeoform to support multi applications.
The routes now looks like : 
c2cgeoform_index: /{application}/{table}
So the admin app segment is not a route_prefix anymore, it is an option of the application registered in c2cgeoportal_admin.routes module.

Reply:
OK, thanks :-)